content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -71,32 +82,50 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
 
     On the product page, you can see an error message on the **Book Reviews** section.
     The message reads: **"Ratings service is currently unavailable."**. The message appears because we
-    switched to use the `v2` subset of the `ratings` service without deploying the MongoDB service.
+    switched to use the `v2` subset of the `ratings` workload without deploying the MongoDB workload.
+
+1. Deploy the MongoDB workload:
+
+    {{< tabset cookie-name="sidecar" >}}
+
+    {{< tab name="Automatic sidecar injection enabled" cookie-value="auto" >}}
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
+    {{< /text >}}
 
-1. Deploy the MongoDB service:
+    {{< /tab >}}
 
-    * To deploy MongoDB in a cluster **with** automatic sidecar injection enabled:
+    {{< tab name="Manual sidecar injection" cookie-value="manual" >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
+    {{< /text >}}
 
-    * To deploy MongoDB in a cluster **without** automatic sidecar injection enabled:
+    {{< /tab >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
-        {{< /text >}}
+    {{< /tabset >}}
 
 1. Go to the Bookinfo product page at `http://$GATEWAY_URL/productpage`.
 
 1. Verify that the **Book Reviews** section shows the reviews.
 
-## Enabling Istio authorization
+## Applying a default `deny-all` policy on the workload
 
-Run the following command to enable Istio authorization for the MongoDB service:
+Run the following command to apply a default `deny-all` policy for the MongoDB workload, the
+command creates a `deny-all` policy that denies all requests to the MongoDB workload:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/rbac-config-on-mongodb.yaml@
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+EOF
 {{< /text >}}
 
 Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).  You should see:
```

Comment:
Please verify that this text is properly indented to ensure the numbered list remains unbroken throughout.